src/it/resources/com/google/checkstyle/test/chapter5naming/rule523methodnames/InputMethodName.java

```diff
@@ -22,6 +24,27 @@ void f() {} // violation 'Method name 'f' must match pattern'
 
   void fO() {} // violation 'Method name 'fO' must match pattern'
 
+  @Test
+  void testing_foo() {}
```

Comment:
Please add similar method but without `@Test` annotation, I think we should expect violation for non-test methods according to the doc:

> Underscores may appear in JUnit **test** method names

Reply:
done, pushed the changes:

https://github.com/checkstyle/checkstyle/blob/29d91cab6fffc7375b24dd7656e0f08ca1b117fb/src/it/resources/com/google/checkstyle/test/chapter5naming/rule523methodnames/InputMethodName.java#L65-L77

though we're checking the JUnit methods and normal methods same way, there's no difference in violations. We are not checking if the method is a test method or not

Reply:
MethodName module from google_checks.xml:

https://github.com/checkstyle/checkstyle/blob/e071377c4d6e049491aef610b04bc106e8a8b6a3/src/main/resources/google_checks.xml#L353-L357